frontend/public/module/k8s/installed-operators.ts

```diff
@@ -0,0 +1,6 @@
+import * as _ from 'lodash-es';
```

Comment:
It's too bad this can't go in the OLM package, but I think it needs to be here since it's reference from table-filters.ts.

I might just inline this in table-filters since we don't want to encourage a lot of OLM code outside of the package, and it's only used in one place.

Reply:
@spadgett Updated! Thanks